internal/server/http/handler/v1/user/handler.go

```diff
@@ -187,9 +183,63 @@ func parseURL(rawURL string) (*url.URL, error) {
 // @Param			limit	query		string	false	"Limit"
 // @Router			/v1/user [get]
 // @Security		Keycloak
-func GetAllUsers(_ service.AuthService) fiber.Handler {
+func GetAllUsers(svc service.AuthService) fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		return c.SendStatus(fiber.StatusNotImplemented)
+		ctx := c.Context()
+
+		users, err := svc.GetAll(ctx)
+		if err != nil {
+			return errorhandler.HandleError(service.NewError(service.InternalError, errors.Wrap(err, "failed to get all users").Error()))
+		}
+		response := make([]entities.UserResponse, len(users))
+
+		for i, user := range users {
+			userResponse := userMapper.FromResponse(user)
+			response[i] = *userResponse
+		}
+
+		return c.Status(fiber.StatusOK).JSON(response)
+	}
+}
+
+// @Summary		Get users by role
+// @Description	Get users by role
+// @Tags			User
+// @Produce		json
+// @Success		200		{object}	entities.UserListResponse
+// @Failure		400		{object}	HTTPError
+// @Failure		500		{object}	HTTPError
+// @Param			page	query		string	false	"Page"
+// @Param			limit	query		string	false	"Limit"
+// @Param			limit	query		string	false	"Role"
+// @Router			/v1/user [get]
+// @Security		Keycloak
+func GetUsersByRole(svc service.AuthService) fiber.Handler {
```

Comment:
Cedrik left on my PR to the vehicles a comment that it would be better if we use for this kind of stuff parameter and not a new endpoint. 

so GetAll but use something like `users?role=tbz`